lm_eval/models/huggingface.py

```diff
@@ -405,9 +407,87 @@ def tokenizer_name(self) -> str:
 
     @property
     def chat_template(self) -> str:
-        if self.tokenizer.chat_template is not None:
-            return self.tokenizer.chat_template
-        return self.tokenizer.default_chat_template
+        """
```

Comment:
We might want to simplify this function. I get that you wanted to make it more comparable to the transformers one but it's always better to aim for self contained/simple code, imo.